addon/components/ember-th/sort-indicator/component.js

```diff
@@ -0,0 +1,57 @@
+import Component from '@ember/component';
+import layout from './template';
+
+import { argument } from '@ember-decorators/argument';
+import { required } from '@ember-decorators/argument/validation';
+import { type } from '@ember-decorators/argument/type';
+import { tagName } from '@ember-decorators/component';
+import { readOnly } from '@ember-decorators/object/computed';
+
+/**
+  The table header cell sort indicator component. This component renders the state of the sort on the column (ascending/descending/none).
+
+  ```hbs
+  <EmberTable as |t|>
+    <t.head @columns={{columns}} as |h|>
+      <h.row as |r|>
+        <r.cell as |columnValue columnMeta|>
+          {{columnValue.name}}
+
+          <EmberThSortIndicator columnMeta=columnMeta>
```

Comment:
I think this should be `<EmberTh::SortIndicator @columnMeta={{columnMeta}} />`

Reply:
👍 I never used that form and couldn't find anywhere in the docs how it's supposed to be.